elevenlabs_api_benchmark.py

```diff
@@ -120,9 +131,43 @@ def truncate_audio_string(audio_string):
         return audio_string[:max_length] + "..."
     return audio_string
 
+# Analyze audio data function
+def analyze_audio(audio_data):
+    # create a Magic object for MIME type detection
+    mime = magic.Magic(mime=True)
+    
+    # detect the MIME type of the audio data
+    file_type = mime.from_buffer(audio_data)
+    audio_format = AUDIO_FORMATS.get(file_type, "Unknown")
```

Comment:
you might also want to print the content-type header too as part of this analysis